examples/react-native/src/features/Authenticator/SignUpAttributesAll/Example.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Button } from 'react-native';
+
+import { Authenticator, useAuthenticator } from '@aws-amplify/ui-react-native';
+import { Amplify } from 'aws-amplify';
+
+import awsconfig from './aws-exports';
+
+Amplify.configure(awsconfig);
+
+function SignOutButton() {
+  const { signOut } = useAuthenticator();
+  return <Button title="Sign Out" onPress={signOut} />;
+}
+
+function App() {
+  return (
+    <Authenticator.Provider>
+      <Authenticator
+        signUpAttributes={[
+          'address',
+          'birthdate',
+          'email',
+          'family_name',
+          'gender',
+          'given_name',
+          'locale',
+          'middle_name',
+          'name',
+          'nickname',
+          'phone_number',
+          'picture',
+          'preferred_username',
+          'profile',
+          'updated_at',
+          'website',
+          'zoneinfo',
+        ]}
```

Comment:
Do we really need an example for this? Feel like a list of attributes/link to the OIDC spec should be sufficient?

Reply:
That makes sense, I thought I was missing something. Will update docs to just list these